problemsets-mat223/modules/module6.tex

```diff
@@ -54,7 +54,49 @@
 	Let $\mathcal V\subseteq \R^2$ be the complete solution to 
 	$x+2y=0$.  Show that $\mathcal V$ is a subspace.
 
-	XXX Finish
+	Let $\vec u = \mat{u_1\\u_2}$, $\vec v = \mat{v_1\\v_2} \in \mathcal V$.
+	This means that 
+	\begin{align*}
+		u_1+2u_2=0 \\
+		v_1+2v_2=0
+	\end{align*}
+	Let $k \in \R$.
```

Comment:
Better to say `Let $k$ be a scalar.` Move this to where you "let" the other variables.